recipes/sail/all/conanfile.py

```diff
@@ -1,5 +1,7 @@
+# pylint: skip-file
 from conan.tools.files import rename
 from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
```

Comment:
```suggestion
from conan.errors import ConanInvalidConfiguration
```
Use the new namespace: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#python-import-statements